app/controllers/users/invitations_controller.rb

```diff
@@ -99,9 +106,6 @@ def invited_user_has_no_org?
 
   # Overrides https://github.com/scambra/devise_invitable/blob/master/app/controllers/devise/invitations_controller.rb#L105
   def invite_params
```

Comment:
Given we don't construct objects with this, is it fine to not make it permitted params?